docs/getting-started/installing/installing-core-linux.md

```diff
@@ -2,119 +2,71 @@
+We will use apt-get, the native Ubuntu package installer, to install the .NET Core SDK. In order to get the package, however, we will need to add a new package feed. The below commands will do this. 
 
 ```console
-mozroots --import --sync
+sudo sh -c 'echo "deb [arch=amd64] http://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
+sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
+sudo apt-get update
+``` 
 
-```
 
-## Installing DNVM
+## Installing the .NET Command Line Interface
 
-You need DNVM as a starting point. DNVM enables you to acquire one or multiple .NET Execution Environments (DNX). DNVM is a shell script and does not require .NET. You can use the below command to install it.
+Installing the actual CLI toolchain is as simple as running:
 
 ```console
-curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
-
+sudo apt-get install dotnet
 ```
 
-## Installing the .NET Core DNX
-
-You first need to acquire the CoreCLR DNX.
-
-```console
-dnvm install latest -r coreclr -u
-
-```
-
-You can see the currently installed DNX versions with `dnvm list`.
-
-```console
-dnvm list
-
-```
-
-```console
-Active Version              Runtime Arch Location             Alias
------- -------              ------- ---- --------             -----
-  *    1.0.0-beta5-11649    coreclr x64  ~/.dnx/runtimes
-
-```
-
-## Using a specific runtime
-
-You can choose which of the installed DNXs you want to use with `dnvm use`, specifying arguments that are similar to the ones used when installing a runtime.
-
-```console
-dnvm use -r coreclr -arch x86 1.0.0-beta5-11649
-Adding ~/.dnx/runtimes/dnx-coreclr-win-x86.1.0.0-beta5-11649/bin
-to process PATH
-
-dnvm list
-
-Active Version              Runtime Arch Location             Alias
------- -------              ------- ---- --------             -----
-  *    1.0.0-beta5-11649    coreclr x64  ~/.dnx/runtimes
-
-```
-
-See the asterisk in the listing above? It’s purpose is to tell you which runtime is now active. “Active” here means that all of the interaction with your projects and .NET Core will use this runtime.
+This will install the package and all of the dependencies. It will also add the toolchain to your $PATH, so they will be available the next time you drop down into the terminal.  
 
 That’s it! You now have the .NET Core runtime installed on your machine and it is time to take it for a spin.
 
 ## Write your App
 
-his being an introduction-level document, it seems fitting to start with a “Hello World” app. Here’s a very simple one you can copy and paste into a CS file in a directory.
+This being an introduction-level document, it seems fitting to start with a “Hello World” app. Here’s a very simple one you can copy and paste into a CS file in a directory.
```

Comment:
Yeah, this is a short fix, I'm hoping that we can replace these documents with the pointer to the "getting started" on the Core website.